OJP/OJP_JourneySupport.xsd

```diff
@@ -357,6 +367,9 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="ServiceTimeGroup"/>
+			<xs:element ref="siri:ArrivalFormationAssignment" minOccurs="0"/>
```

Comment:
Is the cardinality correct? *DepartureFormationAssignment* needs *maxOccurs="unbounded"*.

Reply:
yes.. there might be more than one.